resources/js/components/notification-icon.tsx

```diff
@@ -13,8 +13,7 @@ interface Props {
 }
 
 function format(count: number) {
```

Comment:
remove this function and call formatNumber directly

Reply:
sure